lms/djangoapps/program_enrollments/api/__init__.py

```diff
@@ -27,7 +27,7 @@
     get_program_course_enrollment,
     get_program_enrollment,
     get_provider_slug,
-    get_saml_provider_for_organization,
+    get_saml_providers_for_organization,
```

Comment:
Should we keep the original function to return a single SAML provider? As for which one if there were two, we can select the most recent one. Might introduce complexity, but I want to ask what made you decided on straight update to move all logic around this function to evaluate multiple providers.

Reply:
I'd think in the case where we have multiple (active) providers I would expect both of them to work. If we only return the latest we can't read/write enrollments for any users with a uid not on the new provider.

Reply:
^ by extension, if both work at the same time doing cutovers becomes simpler since it's a non-breaking change to add a new one.

Reply:
Got it. I agree then.